src/python3/Makefile

```diff
@@ -0,0 +1,34 @@
+.ONESHELL:
+SHELL = /bin/bash
+.SHELLFLAGS += -e
+
+PYTHON_VER=3.6.0
+PYTHON_DEB_VER=1
+PYTHON_PNAME=python3.6
+
+MAIN_TARGET = lib$(PYTHON_PNAME)-minimal_$(PYTHON_VER)-$(PYTHON_DEB_VER)_amd64.deb
+DERIVED_TARGETS = lib$(PYTHON_PNAME)-stdlib_$(PYTHON_VER)-$(PYTHON_DEB_VER)_amd64.deb \
+				  lib$(PYTHON_PNAME)_$(PYTHON_VER)-$(PYTHON_DEB_VER)_amd64.deb \
+				  $(PYTHON_PNAME)-minimal_$(PYTHON_VER)-$(PYTHON_DEB_VER)_amd64.deb \
+				  $(PYTHON_PNAME)_$(PYTHON_VER)-$(PYTHON_DEB_VER)_amd64.deb \
+				  lib$(PYTHON_PNAME)-dev_$(PYTHON_VER)-$(PYTHON_DEB_VER)_amd64.deb
+				  #$(PYTHON_PNAME)-dev_$(PYTHON_VER)-$(PYTHON_DEB_VER)_amd64.deb
+
+$(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+	## Obtaining the python3
+	wget http://http.debian.net/debian/pool/main/p/$(PYTHON_PNAME)/$(PYTHON_PNAME)_$(PYTHON_VER).orig.tar.xz -O $(PYTHON_PNAME)_$(PYTHON_VER).orig.tar.xz
```

Comment:
python3.6 is in stretch meaning there will be updates, probably we should upload to the azure storage and download from there.